appveyor.yml

```
@@ -5,15 +5,17 @@ environment:
   global:
     SALT_FILEROOT: '%APPVEYOR_BUILD_FOLDER%'
     SALT_STATE: scap.scan
-    SALT_URL: https://repo.saltproject.io/windows/Salt-Minion-3004.2-1-Py3-AMD64-Setup.exe
+    SALT_URL:  'https://repo.saltproject.io/windows/Salt-Minion-3005.1-5-Py3-AMD64-Setup.exe'
   matrix:
     - SALT_PILLARROOT: '%APPVEYOR_BUILD_FOLDER%\tests\pillar\test-scc'
 
 init:
-  - ps: echo $env:PILLAR_HOME
+  - ps: |
+      echo $env:PILLAR_HOME
 install:
   - ps: |
       $null = `
+        [Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls13"
         (new-object net.webclient).DownloadFile(${env:SALT_URL}, `
         "${env:temp}\salt-minion.exe")
```

Try something like this:

```suggestion
  - ps: [Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls13"
  - ps: |
      $null = `
        (new-object net.webclient).DownloadFile(${env:SALT_URL}, `
        "${env:temp}\salt-minion.exe")
```